src/transformers/models/regnet/modeling_tf_regnet.py

```diff
@@ -194,16 +202,17 @@ def __init__(self, in_channels: int, reduced_channels: int, **kwargs):
         super().__init__(**kwargs)
 
         self.pooler = TFAdaptiveAvgPool2D(output_shape=(1, 1), name="pooler")
-        self.attention = tf.keras.Sequential([
-            tf.keras.layers.Conv2D(filters=reduced_channels, kernel_size=1, activation="relu"),
-            tf.keras.layers.Conv2D(filters=in_channels, kernel_size=1, activation="sigmoid"),
-        ], name="attention")
+        self.attention = [
+            tf.keras.layers.Conv2D(filters=reduced_channels, kernel_size=1, activation="relu", name="attention.0"),
+            tf.keras.layers.Conv2D(filters=in_channels, kernel_size=1, activation="sigmoid", name="attention.2"),
```

Comment:
Since the PyTorch model uses the activations in isolation, we need to skip the layer number.

Reply:
This is a great catch!